server/src/main/java/org/apache/iotdb/db/query/externalsort/ExternalSortJobScheduler.java

```diff
@@ -21,15 +21,15 @@
 
  public class ExternalSortJobScheduler {
 
-  private long queryId = 0;
+  private long taskId = 0;
 
   private ExternalSortJobScheduler() {
 
   }
 
-  synchronized long genQueryId() {
-   queryId++;
-   return queryId;
+  synchronized long genTaskId() {
+   taskId++;
+   return taskId;
```

Comment:
If the use of this class is only to generate a global unique taskId for external sort job, why don't we use a AtomicLong variable, and there is no need to use synchronized, it is too heavy.

Reply:
changed